persistence/sql/migrations/sql/20240425095000000000_identity_credentials_fix_user_handle_index.cockroach.up.sql

```diff
@@ -0,0 +1,6 @@
+DROP INDEX identity_credentials_user_handle_idx;
```

Comment:
I think it would be better to invert this, so first add the index and then remove it. Because then you never have a time where no index exists which leads to very slow passkey logins.

Reply:
Sure, I can do this. Just FYI, we only need this index when logging in with the passkey method when the user only has a webauthn credential, so in the migration case. Usual passkey logins won't even need this index, because there the user handle is the credential identifier.

Reply:
fixed